packages/rx_bloc_cli/example/docs/mfa.md

```diff
@@ -0,0 +1,257 @@
+Define the `MFAEmailVerificationRoute` class extending `GoRouteData` to handle routing.
+Finally, add the new route path in the `RoutesPath` class and update the `createMFAMethodRoute` method to include the new enum case.
+
+
+#### Step 1: Payload - Add a new case in the MFAMethod enum
+```dart
+enum MFAMethod {
+  ...
+  @JsonValue('EmailVerification')
+  emailVerification,
+```
+
+#### Step 2: Payload -  Create a class that extends MFAPayloadRequest and uses the new MFAMethod
+```dart
+class MFAEmailVerificationPayload extends MFAPayloadRequest
+    with EquatableMixin {
+  MFAEmailVerificationPayload();
+
+  @override
+  String get type => MFAMethod.emailVerification.name;
+
+  factory MFAEmailVerificationPayload.fromJson(Map<String, dynamic> json) => MFAEmailVerificationPayload();
+
+  @override
+  Map<String, dynamic> payloadToJson() => {};
+
+  @override
+  List<Object?> get props => [type];
+
+  @override
+  bool? get stringify => true;
+}
+```
+
+#### Step 3: Payload - Add a new case in the MFAMethodRequest class to parse the new request method payload 
+```dart
+MFAPayloadRequest _payloadFromJson(Map<String, dynamic>? json, type) {
+  switch (type) {
+    ...
+    case MFAMethod.emailVerification:
+      return MFAEmailVerificationPayload.fromJson(json);
+  }
+```
+
+#### Step 4: Route - Create a new route class that extends GoRouteData and uses the new mfaMethod
+```dart
+@TypedGoRoute<MFAEmailVerificationRoute>(
+  path: RoutesPath.MFAEmailVerification,
+)
+class MFAEmailVerificationRoute extends GoRouteData implements RouteDataModel {
+  const MFAEmailVerificationRoute(
+    this.transactionId,
+  );
+
+  final String transactionId;
+
+  @override
+  Page<Function> buildPage(BuildContext context, GoRouterState state) =>
+      MaterialPage(
+        key: state.pageKey,
+        child: ....
+      );
+
+  @override
+  String get permissionName => RouteModel.mfa.permissionName;
+
+  @override
+  String get routeLocation => location;
+}
+```
+
+
+#### Step 5: Route - Define the new route path in the RoutesPath class
+```dart
+class RoutesPath {
+  ....
+  static const mfaEmailVerification = '/mfa/email-verification/:transactionId';
+}
+```
+
+
+#### Step 6: Route - Add a new case in the createMFAMethodRoute method
+```dart
+extension MFAMethodX on MFAMethod {
+  RouteDataModel? createMFAMethodRoute(String transactionId) {
+   switch (this) {
+      ...
+      case MFAMethod.emailVerification:
+        return MFAEmailVerificationRoute(transactionId);
+    }
+  }
+```
+
+[mfa_img]: https://raw.githubusercontent.com/Prime-Holding/rx_bloc/feature/auth-matrix-refactoring-2fa/packages/rx_bloc_cli/example/docs/mfa.png
+[mfa_sequence_img]: https://raw.githubusercontent.com/Prime-Holding/rx_bloc/feature/auth-matrix-refactoring-2fa/packages/rx_bloc_cli/example/docs/mfa_sequence.png
+[mfa_c4]: https://raw.githubusercontent.com/Prime-Holding/rx_bloc/feature/auth-matrix-refactoring-2fa/packages/rx_bloc_cli/example/docs/mfa_c4.png
```

Comment:
These links currently return 404, I suppose they will need to be updated to point to `master` after the merge (or a relative path)

Reply:
The links now points to the develop branch. They will be valid once we merge this PR.